docs/split_keyboards.md

```diff
@@ -35,44 +48,92 @@ split = Split(
-longer than 11 characters. Instructions on how to do that are
-[here](https://learn.adafruit.com/welcome-to-circuitpython/renaming-circuitpy).
-For example on NYQUISTL for left and NYQUISTR for the right.
+This is a sane default settings.
+
+But you still can choose to set all that manually. Some reasons would be:
+- Easy both side debug and upload by having both halves connected to usb (and uart between them). Explicit `split_side` and only one `split_target` is needed to avoid each half acting as a `split_target` (duplicating each keypress).
+- Wanting to be explicit about hardware discrepancy between halves (others than just reversing the rows see `split_flip`). Keep in mind that the normal case (interchangeable pcb between sides) do **not** need this. The following code is **not** a guideline, but an extreme example showing the flexibility of kmk (would realistically be applicable only in messy handwired keyboard):
 
-For wired connections you don't need to pass anything. For Bluetooth, remove the `split_side` like this
 ```python
-# Wired
-split = Split()
-# Wireless
-split = Split(split_type=SplitType.BLE)
+from storage import getmount
+
+side = SplitSide.RIGHT if str(getmount('/').label)[-1] == 'R' else SplitSide.LEFT
+
+if side == SplitSide.RIGHT:
+    keyboard.col_pins = ...
+    keyboard.row_pins = ...
+    keyboard.diode_orientation = ...
+else:
+    keyboard.col_pins = ...
+    keyboard.row_pins = ...
+    keyboard.diode_orientation = ...
+
+split = Split(
+    split_side=side,
+    target_left=True,
+    ...
+)
 ```
 
-### RP2040 PIO implementation
+Note: It is best to stay as coherent as possible, but thanks to the `coord_mapping` feature, none of the `col_pins`, `row_pins` and `diode_orientation` **need** to be the same for both side. The only need is for `len(col_pins) * len(row_pins)` to be the same (as it is assumed that both side will handle the same numbers of potential keys.
+
+#### `split_flip`
+If your split keyboard use the **same pcb for both side**, but vertically flipped, Set this to `True`, and `False` otherwise. `True` will means everything are the same for both side except that `row_pins` are reversed (as you flip one of the pcb verticaly).
+
+You will need to update your `coord_mapping` (if you use one) if you change this value.
+
+#### `split_target_left`
+The `split_target` refer to the side that will send keypress over usb.
 
-If you're using RP2040 based board and want to use other pins that the one with hardware support of RX/TX, you can try
-using PIO implementation of two-way wire protocol. Typical use cases for it are premade boards, made with QMK's
-bitbanging protocols in mind.
+If you use EE HANDS (`split_side = None`) this parameter will be automatically overriden (a side connected to a usb host will act as a `split_target`).
+
+If setting this manually, be sure to reflect the actual usb connection and make sure to use the same value for both sides !
```

Comment:
```suggestion
Setting `split_side = None` (similar to EE HANDS in QMK) this parameter will be overriden.
```
* KMK doesn't have EE HANDS. It's a similar feature that just copied the name from QMK.
* I removed some redundant text.